Application/Items/Commands/Create.cs

```diff
@@ -56,7 +56,7 @@ public async Task<Unit> Handle(Command request, CancellationToken cancellationTo
                     }
                 }
 
-                request.Item.Uploader = request.User;
+                request.Item.Uploader = await _context.Users.FindAsync(request.User);
```

Comment:
check the value of request.item.uploader, because I think the wrong data is assigned